File: src/swingjs/plaf/JSRadioButtonMenuItemUI.java
package swingjs.plaf;

import jsjavax.swing.JMenuItem;

public class JSRadioButtonMenuItemUI extends JSRadioButtonUI {
	
	public JSRadioButtonMenuItemUI() {
		super();
		isMenuItem = true;
		allowPaintedBackground = false;
	}
	
	@Override
	protected String getPropertyPrefix() {
		return "RadioButtonMenuItem.";
	}
	
	@Override
	protected void installUIImpl() {
		menuItem = (JMenuItem) c;
		super.installUIImpl();
	}
}

File: site/swingjs/j2s/swingjs/plaf/JSRadioButtonMenuItemUI.js
Clazz.declarePackage ("swingjs.plaf");
Clazz.load (["swingjs.plaf.JSRadioButtonUI"], "swingjs.plaf.JSRadioButtonMenuItemUI", null, function () {
c$ = Clazz.declareType (swingjs.plaf, "JSRadioButtonMenuItemUI", swingjs.plaf.JSRadioButtonUI);
Clazz.overrideMethod (c$, "getDOMObject", 
function () {
return this.getButtonObject ("radio");
});
Clazz.overrideMethod (c$, "getPropertyPrefix", 
function () {
return "RadioButtonMenuItem.";
});
});
